test/DurableTask.SqlServer.Tests/DurableTask.SqlServer.Tests.csproj

```diff
@@ -21,4 +21,10 @@
     <ProjectReference Include="..\..\src\DurableTask.SqlServer\DurableTask.SqlServer.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="DatabaseBackups\DurableDB-v1.0.0.bak.zip">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
I noticed we committed this file. Was this intentional? I would assume that perhaps we only want this generated and cleaned up at test time.

I just worry about keeping `.zip` files in the commit history, which sometimes have weird diffs over time.

Reply:
Yes, this was intentional. I suppose we could generate it during the test, but that would require a lot more test code, not to mention other complexities once we do upgrade tests for multiple versions. These zip files should never change after being generated. The idea is that this backed up database (and any other versions we add here) should be _forever_ compatible with the SQL provider (or at least until we release a v2.x version).